packages/testutils/src/__tests__/mockEndpoint.spec.ts

```diff
@@ -368,3 +368,50 @@ it('accepts request instance', async () => {
     new Response(null, { status: 500 }),
   );
 });
+
+describe('warning unmocked endpoint', () => {
```

Comment:
we don't need `describe` here, `it('warns about not matched endpoints')` will be enough.